README.md

```diff
@@ -9,3 +9,7 @@ drunk oni irc robot
 ## Dependencies
 * appdirs
 * Twisted with SSL support
+  * pyopenssl
+  * pypiwin32 (on Windows)
+* PyYAML (`modules/util.py`)
+* humanize, pytz, parsedatetime
```

Comment:
I would probably specify that these are plugin-specific dependencies and not strictly needed